### PR TITLE
Todo: Recycle bin

### PR DESCRIPTION
**Pull request by SmallMistake**
The features page of the documentation website said you all wanted to do a recycling page. So, I made this one. Let me know if you think it needs any more info or should be moved to a different spot.

### CODE REVIEW

File: content/features/content-management/recycle-bin/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+uid: recycle-bin
+locale: en
+title: Recycle Bin
+dnnversion: 09.10.00
+---
+# Recycle Bin
+# Overview
+Just like your computer, when you delete content from your website it does not instantly get deleted from the system. Instead it is sent to a recycling bin that can be found under `Content` - `Recycle Bin`. There you will be able to view, restore, or permanently delete content that has been sent to the bin. If you want to empty your recycling bin, you can press the `Empty Recycle Bin` button. 
+
+![RecycleBin](https://user-images.githubusercontent.com/44900498/157530166-0652335d-560c-43eb-b94e-342ce46d1410.PNG)
```
Comment (valadas) [inline content/features/content-management/recycle-bin/index.md]: Images need to but part of the repository and not be github user images

**Review by valadas — CHANGES_REQUESTED** [review]
Images need to be part of the repository here it look like the image was probably drag-and-dropped in github

File: content/features/content-management/recycle-bin/index.md (hunk 1 above)
Comment (SmallMistake) [reply content/features/content-management/recycle-bin/index.md]: Okay, I just committed the fix for this one and the commits at the top seem to show it working. Thanks for pointing this out.

**Review by valadas — APPROVED** [review]
Awesome, thanks

**Review by bdukes — APPROVED** [review]

**Review by david-poindexter — APPROVED** [review]

**Pull request merged**